File: TrueFalseQuiz/app/src/main/java/com/comp1601/truefalsequiz/MainActivity.java
package com.comp1601.truefalsequiz;

import androidx.appcompat.app.AppCompatActivity;

import android.os.Bundle;
import android.view.View;
import android.widget.Button;
import android.widget.Toast;

import java.util.ArrayList;

public class MainActivity extends AppCompatActivity {
    private Button mYesButton;
    private Button mNoButton;


    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        //inflate buttons
        mYesButton = findViewById(R.id.yes_button);
        mNoButton = findViewById(R.id.no_button);

        ArrayList<Question> questions = new ArrayList<>();
        questions.add(new Question(getString(R.string.question1)));
        questions.add(new Question(getString(R.string.question2)));
        questions.add(new Question(getString(R.string.question3)));
        questions.add(new Question(getString(R.string.question4)));
        questions.add(new Question(getString(R.string.question5)));

        for (Question q: questions){
            System.out.println(q.getQuestion());
            System.out.println(q.getAnswer());
        }


        mYesButton.setOnClickListener(v -> {
            //handle yes button click
            String s = getString(R.string.yes_button_label);
            System.out.println("YOU PRESSED AA" + s);


            Toast.makeText(MainActivity.this, s, Toast.LENGTH_SHORT).show();
        });

        mNoButton.setOnClickListener(v -> {
            //handle no Button click
            String s = getString(R.string.no_button_label);
            System.out.println("YOU PRESSED " + s);
            Toast.makeText(MainActivity.this, s, Toast.LENGTH_SHORT).show();

        });

    }
}

File: Multiple_Choice_Quiz_App/app/src/main/java/com/comp1601/multiple_choice_quiz_app/MainActivity.kt
package com.comp1601.multiple_choice_quiz_app

import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import android.widget.Button
import android.widget.TextView

class MainActivity : AppCompatActivity() {




    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

        val myQuestion:TextView = findViewById(R.id.txtQuestion)
        val btnMultipleChoiceA:Button = findViewById(R.id.btnMultipleChoiceA)
        val btnMultipleChoiceB:Button = findViewById(R.id.btnMultipleChoiceB)
        val btnMultipleChoiceC:Button = findViewById(R.id.btnMultipleChoiceC)
        val btnMultipleChoiceD:Button = findViewById(R.id.btnMultipleChoiceD)
        val btnMultipleChoiceE:Button = findViewById(R.id.btnMultipleChoiceE)


        var currentQuestionIndex = 0

        //list of all questions in test
        val questions:ArrayList<Question> = ArrayList()

        questions.add(Question(getString(R.string.question1)))

        //show first question
        myQuestion.text = questions[currentQuestionIndex].getQuestion()



        btnMultipleChoiceA.setOnClickListener{

        }









    }
}
